aytests/cms.xml

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+
+  <add-on>
+    <add_on_products config:type="list">
+      <listentry>
+        <media_url>http://download.opensuse.org/repositories/systemsmanagement:/saltstack:/products/SLE_12_SP2/</media_url>
```

Comment:
Because there is currently no SP3 branch for that OBS project. We will switch if it is available.